.github/actions/silences-report-expired/silences-report-expired.sh

```diff
@@ -7,13 +7,16 @@ KUSTOMIZATION_FILENAME="kustomization.yaml"
 
 DEFAULT_EXPIRATION="14 days"
 COMMIT_MESSAGE_PREFIX="remove expired silence - "
-PULL_REQUEST_BODY="This pull request was automatically created using the $(basename $0) script.
+PULL_REQUEST_BODY_EXPIRED="This pull request was automatically created using the $(basename $0) script.
 
-If you are assigned for review this means you created a silence which is now expired, based on the \`valid-until\` annotation, and has **been removed from AlertManager**.
+If you are assigned for review, this means you created a silence which is now expired based on the \`valid-until\` annotation, and has **been removed from AlertManager**.
```

Comment:
Maybe we emphasis the `now expired` part as you did it for the `expiring soon` in the other body.

```suggestion
If you are assigned for review, this means you created a silence which is **now expired** based on the \`valid-until\` annotation, and has **been removed from AlertManager**.
```